utils/data-ingestor.js

```diff
@@ -123,19 +123,26 @@ class DataIngestor {
     const specialType = updatedRow.get('specialType');
     if (specialType === undefined) return updatedRow;
 
+    // unset percent and percent_m
+    updatedRow = this.unsetPercents(updatedRow);
```

Comment:
unset these values because they are being recalculated later anyways OR will be "undefined" if the type is noPercentOnly?

Reply:
i refactored how this is being done, hopefully to make it more clear. But unset them b/c they should just be undefined at the end of the day. `noPercentOnly` type means "special, but only b/c they should not have percent values. I.e. they don't need estimate & moe recalculated. but see updated.